sdk/platform/README.md

```diff
@@ -0,0 +1,38 @@
+# Azure SDK Platform
+
+The Azure SDK platform provides two abstractions with some built-in implementations that the caller can use or override with their own behavior. The first one is a transport adapter with helpers to allow clients to communicate with Azure. The second one is operating system specific implementations for the small set of features  that the Azure SDK needs that are not available as part of the C99 standard library (such as thread sleep).
+
+## HTTP Transport Adapter
+
+The Azure SDK has its own HTTP request and response structures. For this reason, an HTTP transport adapter is required to be implemented to use the Azure SDK HTTP request and response as input for any HTTP stack (like libcurl or win32). The library allows clients to communicate with Azure.
```

Comment:
> for any HTTP stack (like libcurl or win32)

What does `win32` mean here?

This wasn't addressed earlier:
> Did you mean `win32` or something else, like `WinINet`?

Reply:
well, we call it win32.  Honestly I am not really sure about the proper way to call the Windows HTTP stack.
Making a quick search, my understanding is that WinINet is part of win32.
And there's also WinHTTP within win32 API as a way to use HTTP stack.
https://docs.microsoft.com/en-us/windows/win32/wininet/about-wininet

So, I believe we are OK saying win32 as a general Windows API to be used for HTTP.  What do you think?

Reply:
> So, I believe we are OK saying win32 as a general Windows API to be used for HTTP. What do you think?

Sounds good Thanks for clarifying.